src/main/java/seedu/address/model/common/Name.java

```diff
@@ -16,7 +16,7 @@ public class Name {
      * The first character of the address must not be a whitespace,
      * otherwise " " (a blank string) becomes a valid input.
      */
-    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+    public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum}, ]*";
```

Comment:
Is this supposed to be the correct regex? The company name uses a different regex as it has different conditions

Reply:
Alright let's discuss it tomorrow

Reply:
As discussed, considering alnum and comma will cover most of the use case, we will proceed the this simpler implementation for now.